### PR TITLE
Quadrat: Add pagination arrows through block settings, not in CSS

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Now that https://github.com/WordPress/gutenberg/pull/33656 has merged we can set pagination arrows in block attributes. This is better for RTL and customization.

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
This works as intended! Thank you :D 

Just two things: we should do the same for Geologist and I think we shouldn't merge this on trunk since this will only work on 11.5. Now that we are close to getting these themes on dotcom we should decide on a strategy when we change things that have been recently merged into GB.

**Review by MaggieCabrera — APPROVED** [review]

**Comment by jeffikus** [comment]
Great work on this. Solid example of identifying something in a theme, contributing back to Gutenberg, then implementing in the theme! High five! :) 🙌

**Comment by pbking** [comment]
This is working well on wpcom now that 11.5 has been installed.  Removing the DNM tag for inclusion in the next deployment.

**Pull request merged**
